### DISCUSSION TITLE
i did not received my student developer pack

### DISCUSSION THREAD

**Discussion by skaduhs5232** [community/community#145236] [GitHub Education]
### Hi everyone,

I applied and I believe I was accepted into the GitHub Student Program, but to this day I have not received benefits such as GitHub Copilot, what should I do?
![image](https://github.com/user-attachments/assets/84fbef24-ed5a-4a2c-aade-d592686e5009)

**Comment by davevad93** [comment]
Hi @skaduhs5232 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @skaduhs5232 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
